frontend/src/util/Selector.re

```diff
@@ -110,4 +110,6 @@ let addFirst = (item: 'a, selector: t('a, 'b)): t('a, 'b) => {...selector, befor
 let addLast = (item: 'a, selector: t('a, 'b)): t('a, 'b) => {...selector, after: selector.after @ [item]};
 let selected = (selector: t(_, 'a)): 'a => selector.selected;
 let canNext = ({after}: t(_, _)): bool => ListUtils.notEmpty(after);
-let canPrevious = ({before}: t(_, _)): bool => ListUtils.notEmpty(before);
\ No newline at end of file
+let canPrevious = ({before}: t(_, _)): bool => ListUtils.notEmpty(before);
+let allCount = ({before, after}): int => List.length(before) + List.length(after) + 1;
```

Comment:
If we have `{before: [1, 2], selected: None, after: [3, 4]}: Selector.Optional.t`, `allCount` shoudnl't return 5 but 4. You should write function `allCount` in `Required` module.